docs/static_site/src/pages/community/code_review.md

```diff
@@ -157,5 +157,10 @@ changes. Code owner can decide if the code can be merged in case by case
 if some of the reviewers did not respond in time(e.g. a week) and
 existing reviews are sufficient.
 
+Get Started
+-----------
+
+{% Checkout the following [PRs that need review](https://github.com/apache/incubator-mxnet/labels/pr-awaiting-review)" %}
```

Comment:
actually this is causing an error: 

`[2020-10-14T23:15:23.685Z]   Liquid Exception: Liquid syntax error (line 155): Unknown tag 'Checkout' in pages/community/code_review.md`

Reply:
This is a regular markdown link so you don't need `{%`. also we can use apache/mxnet instead of incubator-mxnet to be future-proof

Reply:
Wow. I didn't know apache/mxnet redirects to apache/incubator-mxnet! 2+ years and still :) 
More we learn!